apps/settings/elements/wifi.html

```diff
@@ -47,6 +47,15 @@ <h2 data-l10n-id="availableNetworks"> Available Networks </h2>
         <h2 data-l10n-id="advancedSettings"> Advanced Settings </h2>
       </header>
       <ul>
+        <li id="wifi-sleepMode" class="hint">
+          <label class="pack-switch">
+          <input type="checkbox" name="wifi.sleepMode" data-state="off">
+          <span data-l10n-id="wifi-off-during-sleep">Wi-Fi Off during Sleep</span>
```

Comment:
Based on the latest l10n guideline, the text context should not be included in the html file.